test/cli_error/failure.js

```diff
@@ -47,4 +47,70 @@ describe('Test CLI', () => {
             });
         });
     });
+
+    it('passes rejection to flags.onUnhandledRejection handler', (flags) => {
```

Comment:
I figured it's best to keep all these tests together so that `// covered by test/cli_error/failure.js` comment for disabled coverage remains true?